lib/src/request.dart

```diff
@@ -13,14 +12,11 @@ import 'message.dart';
 import 'util.dart';
 
 /// A callback provided by a Shelf handler that's passed to [Request.hijack].
-@Deprecated("Will be removed in shelf 0.7.0.")
-typedef void HijackCallback(
-    Stream<List<int>> stream, StreamSink<List<int>> sink);
+typedef void _HijackCallback(StreamChannel<List<int>> stream);
```

Comment:
Delete this and use the inline syntax in `_OnHijack.run` – right?

Reply:
Yup good call. We still need _OnHijackCallback since it's used as a class field and we don't want to use the new function type syntax yet.